docs/introduction/exchanges.md

```diff
@@ -1,10 +1,46 @@
-# Exchange
+# Exchange and Transfer
 
 This vocabulary subset looks at exchanges of resources from an independent or neutral viewpoint (not the viewpoint of one of the Agents in the exchange). For example, from one Agent's viewpoint, the exchange may be a Purchase, from the other Agent's viewpoint, it might be a Sale. From the neutral viewpoint, it is an exchange of resources, with usually at least two flows of resources, one from each direction. So for example, the seller might give some goods to the buyer, and the buyer might give some money to the seller. Or in a barter exchange, one agent might give the other some books, and the other agent might compensate with some cookies.
 
 This differs from (for example) the [Good Relations Conceptual Model](http://wiki.goodrelations-vocabulary.org/Documentation/Conceptual_model), which we like and want to intertrade with. But Good Relations assumes Compensation in the form of money; the compensation itself is not a separate promise; and the actual flow events are not part of the model. (That's not a criticism. Good Relations has different goals, and a more minimal model makes sense for them.)
 
 In ValueFlows, we want to track not only the offers and promises, but also the actual flows of resources in networks, in all directions. And we support exchanges that don't involve money as well as those that do.
 
-We also support non-reciprocal one-way transfers, such as in a gift economy.
+We also support non-reciprocal one-way transfers, such as in a gift economy.  An exchange implies at least two transfers.
+
+### Transfers
+
+We want to emphasize that although ownership is the primary right transferred in the current economic system (along with renting or leasing), we seek to focus on transfers not necessarily governed by the legal context of property ownership.  This puts the focus on economic coordination.
+
+For example, perhaps someone has many apple trees, and plans on pressing apple cider. Another agent has an apple press and agrees to exchange use of the press (a resource) for a portion of the apple cider.  The use of the press involves some rights (to use the press for some period of time) and responsibilities (to not run it beyond its capacity and to clean it up before returning it), and the portion of apple cider is transferred in exchange.
+
+Or in a library, a book can be checked out for a period of time.  The agent who checks it out is responsible for caring for it and returning it on time, and can read it, or use it however they want.
+
+Or let's say that a community has farmland and equipment held in common.  The community transfers that to some farmers to use and take care of.  The community also transfers seeds every year to the farmers, enough to grow the food the community needs.  During the year, the harvests are distributed (transferred) to the community members for their consumption.  In reciprocity, the community provides for other needs of the farmers.
```

Comment:
This one sounds like diving into some more elaborate *access control* / *capabilities*. So far it seems like we had an agent which owns something and this agent can make any kind of *Commitment* on this thing - *use*, *consume* etc.

I this use case, what kinds of *Commitments* this farmer can make on that equipment. Once again what kinds of rights and responsiblities community *transfers* to this farmer?

Reply:
The farmer can use the equipment all year to work the farm.  No commitments are needed as far as I can see.  I agree there could be complexity in the agreement, but I think that is out of scope for VF.  I explain it some of it here because I'm trying to explain we don't have to think about ownership.